dbld/images/helpers/packages.manifest

```diff
@@ -250,6 +254,8 @@ lsof                            [devshell]
 netcat                          [devshell]
 python-dbg                      [devshell]
 python3-dbg                     [devshell]
+python3-pip                     [devshell]
```

Comment:
there's already a python3-pip line somewhere else in that file.

Reply:
Good catch, thanks, fixed :)